profiles/common/modules/features/nexteuropa_newsroom/theme/related-content.tpl.php

```diff
@@ -0,0 +1,70 @@
+<?php
```

Comment:
After renaming this file to a more specific one please replace all of the php which are not printing pure variables to a template/preprocess function and make this tpl logic-independent.